src/GridFS/WritableStream.php

```diff
@@ -136,6 +136,8 @@ public function __construct(CollectionWrapper $collectionWrapper, string $filena
             '_id' => $options['_id'],
             'chunkSize' => $this->chunkSize,
             'filename' => $filename,
+            'length' => 0,
+            'uploadDate' => new UTCDateTime(0),
```

Comment:
I added these to avoid differences in file contents between saved and unsaved files.

Reply:
It seems wrong to set a date to unix epoch arbitrarily.
It cannot be `null`?

Reply:
Updated: the codec now also supports `null` values.